xml/containers-basics.xml

```diff
@@ -29,24 +34,29 @@
   </listitem>
   <listitem>
    <para>
-    Containers provide near-native performance. Depending on the runtime, a
-    container can use the host kernel directly, thus minimizing overhead.
+     Containers provide near-native performance. As the kernel overhead is low compare to
```

Comment:
There is nearly no kernel overhead involved when running a VM, afaik it's mostly the hypervisor (which in the case of KVM also happens to live in the kernel :wink:) for virtualization or by the emulation itself, when running on foreign architectures (although Apple's M1 proves us wrong here again…).